packages/lambda-tiler/src/routes/health.ts

```diff
@@ -0,0 +1,65 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import PixelMatch = require('pixelmatch');
+import { Epsg, Tile } from '@basemaps/geo';
+import { LambdaHttpResponse, LambdaContext, HttpHeader } from '@basemaps/lambda';
+import { LogConfig } from '@basemaps/shared';
+import { ImageFormat } from '@basemaps/tiler';
+import { PNG } from 'pngjs';
+import { tile } from './tile';
+
+export function getExpectedTileName(projection: Epsg, tile: Tile, format: ImageFormat): string {
+    return path.join(
+        __dirname,
+        '..',
+        '..',
+        `static/expected_tile_${projection.code}_${tile.x}_${tile.y}_z${tile.z}.${format}`,
+    );
+}
+
+export const TestTiles = [
+    { projection: Epsg.Google, format: ImageFormat.PNG, testTile: { x: 30, y: 30, z: 6 } },
+    { projection: Epsg.Google, format: ImageFormat.PNG, testTile: { x: 0, y: 1, z: 1 } },
+    { projection: Epsg.Google, format: ImageFormat.PNG, testTile: { x: 15, y: 10, z: 4 } },
+];
+
+/**
+ * Health request get health TileSets and validate with test TileSets
+ * - Valid response from get heath tile request
+ * - Valid tile get from the request
+ *
+ * @throws LambdaHttpResponse for failure health test
+ */
+export async function Health(): Promise<LambdaHttpResponse> {
+    for (let i = 0; i < TestTiles.length; i++) {
+        const projection = TestTiles[i].projection;
+        const testTile = TestTiles[i].testTile;
+        const format = TestTiles[i].format;
+        const path = `/v1/tiles/health/EPSG:${projection}/${testTile.z}/${testTile.x}/${testTile.y}.${format}`;
+        const ctx: LambdaContext = new LambdaContext(
+            {
+                requestContext: null as any,
+                httpMethod: 'get',
+                path: path,
+                body: null,
+                isBase64Encoded: false,
+            },
+            LogConfig.get(),
+        );
+
+        // Get the test tiles to compare
+        const response = await tile(ctx);
+        if (response.status != 200) return new LambdaHttpResponse(response.status, response.statusDescription);
+        const resImgBuffer = PNG.sync.read(response.body as Buffer);
+        const testTileName = getExpectedTileName(projection, testTile, format);
+        const testTileFile = fs.readFileSync(testTileName);
```

Comment:
We should cache these file reads, I also wonder how well this will work when packaged.

how big are these tiffs can we just base64 them and store them as a file?